addon/utils.js

```diff
@@ -7,6 +7,12 @@ export function isGeneratorIterator(iter) {
           typeof iter['throw']  === 'function');
 }
 
+export function isEventedObject(c) {
+  return (c &&
+          typeof c.one === 'function' &&
+          typeof c.off === 'function');
```

Comment:
one interesting side effect of this is that you'll get a false positive with jQuery selectors, so you could do `yield waitForEvent($(document), 'click');`

I think this is desirable :) and is something I used in my original [EC presentation](https://github.com/machty/ec-prezo/blob/master/app/application/controller.js#L108) (though it was through a `subscribe()` interface meant for observable integration rather than a one-off "subscribe-to-a-single-event")